file.go

```diff
@@ -33,7 +37,16 @@ func downloadGithubZipFile(gitHubCommit GitHubCommit, gitHubToken string, instan
 		return zipFilePath, wrapError(err)
 	}
 
-	resp, err := httpClient.Do(req)
+	for retries +=1; retries > 0; retries -= 1 {
+		resp, err := httpClient.Do(req)
+		if err != nil {
+			fmt.Printf("Error encountered downloading from %s: %s; continuing", req, err)
+			time.Sleep(retrySleepSeconds * time.Second) // Effective linear backoff
+		}
```

Comment:
This is missing a `break` statement to exit the loop if `err == nil`.

Reply:
Hahahaha. Yes, thanks.

Reply:
I think it might be easier if you wrap this routine in a function. If you switch to using a function, you can do a `return resp, nil` to break the loop.